timber/src/test/java/timber/log/TimberJavaTest.java

```diff
@@ -1,16 +1,16 @@
 package timber.log;
 
-import org.junit.Test;
-
 import static org.junit.Assert.fail;
 
+import org.junit.Test;
```

Comment:
nit: Square code style doesn't seem to think this is needed.

Reply:
It's probably my code style moving things around. I can fix it later today.

Reply:
Done